extensions/mssql/src/schemaCompare/schemaCompareService.ts

```diff
@@ -33,7 +33,10 @@ export class SchemaCompareService implements mssql.ISchemaCompareService {
 	}
 
 	public schemaCompare(operationId: string, sourceEndpointInfo: mssql.SchemaCompareEndpointInfo, targetEndpointInfo: mssql.SchemaCompareEndpointInfo, taskExecutionMode: azdata.TaskExecutionMode, deploymentOptions: mssql.DeploymentOptions): Thenable<mssql.SchemaCompareResult> {
-		const params: contracts.SchemaCompareParams = { operationId: operationId, sourceEndpointInfo: sourceEndpointInfo, targetEndpointInfo: targetEndpointInfo, taskExecutionMode: taskExecutionMode, deploymentOptions: deploymentOptions };
+		// Copying deploymentOptions into new variable and updating optionsMapTable like this will not affect scmp save
+		let options = { ...deploymentOptions };
```

Comment:
const

Reply:
This should be let only right, as these options are being updated in the below line.